nl_server/model/sentence_transformer.py

```diff
@@ -35,14 +35,8 @@ def __init__(self, model_info: LocalModelConfig):
     if model_info.gcs_folder:
       logging.info(f'Downloading tuned model from: {model_info.gcs_folder}')
       model_path = gcs.download_folder(model_info.gcs_folder)
```

Comment:
Maybe skip the if condition, since model_info.gcs_folder is a required parameter now?

Reply:
Done